### PR TITLE
planner: add docs for some variables

### PR DESCRIPTION
**Pull request by qw4990**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v6.2 (TiDB 6.2 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: system-variables.md [hunk 1]
```diff
@@ -335,6 +335,16 @@ This variable is an alias for `last_insert_id`.
...
 
+### max_prepared_stmt_count
+
+- Scope: GLOBAL
+- Persists to cluster: Yes
+- Type: Integer
+- Default value: `-1`
+- Range: `[-1, 1048576]`
+- Indicates the maximum number of [`PREPARE` statement](/sql-statements/sql-statement-prepare.md) of a session.
+- `-1` means no limitation.
```
Comment (songrijie) [inline system-variables.md:346]: Could you describe what happens if the the number exceeds?

File: system-variables.md [hunk 2]
```diff
@@ -1724,6 +1743,34 @@ For a system upgraded to v5.0 from an earlier version, if you have not modified
...
 
+### tidb_opt_cartesian_bcj
+
+- Scope: SESSION | GLOBAL
+- Persists to cluster: YES
+- Type: Integer
+- Default value: `1`
+- Range: `[0, 2]`
+- Indicates whether to allow broadcast CARTESIAN join. 
+- `0` means that it's not allowed, `1` means that it's allow based on `tidb_broadcast_join_threshold_count`, and `2` means it's always allowed even if the table size exceeds the threshold.
```
Comment (songrijie) [inline system-variables.md:1754]: Typo: "that it's allow" -> "that it's allow**ed**"

**Comment by songrijie** [comment]
Others LGTM

File: system-variables.md (hunk 2 above)
Comment (qw4990) [reply system-variables.md]: fixed

File: system-variables.md (hunk 1 above)
Comment (qw4990) [reply system-variables.md:346]: OK, a new example was added.

**Review by chrysan — APPROVED** [review]

File: system-variables.md [hunk 3]
```diff
@@ -1053,6 +1085,15 @@ Constraint checking is always performed in place for pessimistic transactions (d
...
 - For a cluster upgraded from a version earlier than v6.1.0, the default value is still `TRUE`.
 
+### tidb_enable_ordered_result_mode
+
+- Scope: SESSION | GLOBAL
+- Persists to cluster: Yes
+- Default value: `OFF`
+- Indicates whether to sort the final output result automatically.
+- For example, if enable this feature, TiDB will process `select a, max(b) from t group by a` as `select a, max(b) from t group by a order by a, max(b)`.
+
```
Comment (TomShawn) [inline system-variables.md:1095]:
```suggestion
```

File: system-variables.md [hunk 4]
```diff
@@ -1746,6 +1815,33 @@ For a system upgraded to v5.0 from an earlier version, if you have not modified
...
+- Indicates the CPU cost of TiDB to process one row. It's used by the [Cost Model](/cost-model.md).
+
+### tidb_opt_desc_scan_factor
+
+- Scope: SESSION | GLOBAL
+- Persists to cluster: YES
+- Type: Float
+- Range: `[0, 2147483647]`
+- Default value: `3.0`
+- Indicates the cost of TiKV to scan one row from the disk in descending order. It's used by the [Cost Model](/cost-model.md).
```
Comment (TomShawn) [inline system-variables.md:1834]:
```suggestion
- Indicates the cost for TiKV to scan one row from the disk in descending order. This variable is internally used in the [Cost Model](/cost-model.md).
```

File: system-variables.md [hunk 5]
```diff
@@ -1746,6 +1815,33 @@ For a system upgraded to v5.0 from an earlier version, if you have not modified
...
+
+### tidb_opt_disk_factor
+
+- Scope: SESSION | GLOBAL
+- Persists to cluster: YES
+- Type: Float
+- Range: `[0, 2147483647]`
+- Default value: `1.5`
+- Indicates the I/O cost of TiDB to read/write one byte from/to the temporary disk. It's used by the [Cost Model](/cost-model.md).
```
Comment (TomShawn) [inline system-variables.md:1843]:
```suggestion
- Indicates the I/O cost for TiDB to read or write one byte of data from or to the temporary disk. This variable is internally used in the [Cost Model](/cost-model.md).
```

File: system-variables.md [hunk 6]
```diff
@@ -1825,6 +1921,15 @@ mysql> desc select count(distinct a) from test.t;
...
 - If the value of the Limit or TopN operator is smaller than or equal to this threshold, these operators are forcibly pushed down to TiKV. This variable resolves the issue that the Limit or TopN operator cannot be pushed down to TiKV partly due to wrong estimation.
 
+### tidb_opt_memory_factor
+
+- Scope: SESSION | GLOBAL
+- Persists to cluster: YES
+- Type: Float
+- Range: `[0, 2147483647]`
+- Default value: `0.001`
+- Indicates the memory cost of TiDB to store one row. It's used by the [Cost Model](/cost-model.md).
```
Comment (TomShawn) [inline system-variables.md:1931]:
```suggestion
- Indicates the memory cost for TiDB to store one row. This variable is internally used in the [Cost Model](/cost-model.md).
```

File: system-variables.md [hunk 7]
```diff
@@ -1833,6 +1938,15 @@ mysql> desc select count(distinct a) from test.t;
...
 - Specifies whether to allow using outer table as the build side in outer join for TiFlash.
 
+### tidb_opt_network_factor
+
+- Scope: SESSION | GLOBAL
+- Persists to cluster: YES
+- Type: Float
+- Range: `[0, 2147483647]`
+- Default value: `1.0`
+- Indicates the net cost of transferring 1 byte through the network. It's used by the [Cost Model](/cost-model.md).
```
Comment (TomShawn) [inline system-variables.md:1948]:
```suggestion
- Indicates the net cost of transferring 1 byte of data through the network. This variable is internally used in the [Cost Model](/cost-model.md).
```

File: system-variables.md [hunk 8]
```diff
@@ -1874,6 +1988,24 @@ explain select * from t where age=5;
...
 - Specifies whether to allow the optimizer to push `Projection` down to the TiKV or TiFlash coprocessor.
 
+### tidb_opt_scan_factor
+
+- Scope: SESSION | GLOBAL
+- Persists to cluster: YES
+- Type: Float
+- Range: `[0, 2147483647]`
+- Default value: `1.5`
+- Indicates the cost of TiKV to scan one row from the disk in ascending order. It's used by the [Cost Model](/cost-model.md).
```
Comment (TomShawn) [inline system-variables.md:1998]:
```suggestion
- Indicates the cost for TiKV to scan one row of data from the disk in ascending order. This variable is internally used in the [Cost Model](/cost-model.md).
```

File: system-variables.md [hunk 9]
```diff
@@ -1874,6 +1988,24 @@ explain select * from t where age=5;
...
+
+### tidb_opt_seek_factor
+
+- Scope: SESSION | GLOBAL
+- Persists to cluster: YES
+- Type: Float
+- Range: `[0, 2147483647]`
+- Default value: `20`
+- Indicates the start-up cost of TiDB to request data from TiKV. It's used by the [Cost Model](/cost-model.md).
```
Comment (TomShawn) [inline system-variables.md:2007]:
```suggestion
- Indicates the start-up cost for TiDB to request data from TiKV. This variable is internally used in the [Cost Model](/cost-model.md).
```

File: system-variables.md [hunk 10]
```diff
@@ -1053,6 +1085,15 @@ Constraint checking is always performed in place for pessimistic transactions (d
 - Since v6.1.0, the [Join Reorder](/join-reorder.md) algorithm of TiDB supports Outer Join. This variable controls the support behavior, and the default value is `ON`.
 - For a cluster upgraded from a version earlier than v6.1.0, the default value is still `TRUE`.
 
+### tidb_enable_ordered_result_mode
+
+- Scope: SESSION | GLOBAL
+- Persists to cluster: Yes
+- Default value: `OFF`
+- Indicates whether to sort the final output result automatically.
```
Comment (TomShawn) [inline system-variables.md:1093]:
```suggestion
- Specifies whether to sort the final output result automatically.
```

File: system-variables.md [hunk 11]
```diff
@@ -1053,6 +1085,15 @@ Constraint checking is always performed in place for pessimistic transactions (d
...
+- For example, if enable this feature, TiDB will process `select a, max(b) from t group by a` as `select a, max(b) from t group by a order by a, max(b)`.
```
Comment (TomShawn) [inline system-variables.md:1094]:
```suggestion
- For example, with this variable enabled, TiDB processes `SELECT a, MAX(b) FROM t GROUP BY a` as `SELECT a, MAX(b) FROM t GROUP BY a ORDER BY a, MAX(b)`.
```

File: system-variables.md [hunk 12]
```diff
@@ -1724,6 +1764,34 @@ For a system upgraded to v5.0 from an earlier version, if you have not modified
...
 
+### tidb_opt_cartesian_bcj
+
+- Scope: SESSION | GLOBAL
+- Persists to cluster: YES
+- Type: Integer
+- Default value: `1`
+- Range: `[0, 2]`
+- Indicates whether to allow the broadcast CARTESIAN join. 
+- `0` means that the broadcast CARTESIAN join is not allowed. `1` means that it is allowed based on [`tidb_broadcast_join_threshold_count`](#tidb_broadcast_join_threshold_count-new-in-v50). `2` means that it is always allowed even if the table size exceeds the threshold.
```
Comment (TomShawn) [inline system-variables.md:1775]:
```suggestion
- `0` means that the broadcast CARTESIAN join is not allowed. `1` means that it is allowed based on [`tidb_broadcast_join_threshold_count`](#tidb_broadcast_join_threshold_count-new-in-v50). `2` means that it is always allowed even if the table size exceeds the threshold.
- This variable is internally used in TiDB. It is not recommended to modify its value.
```

File: system-variables.md [hunk 13]
```diff
@@ -1724,6 +1764,34 @@ For a system upgraded to v5.0 from an earlier version, if you have not modified
...
+
+### tidb_opt_concurrency_factor
+
+- Scope: SESSION | GLOBAL
+- Persists to cluster: YES
+- Type: Float
+- Range: `[0, 2147483647]`
+- Default value: `3.0`
+- Indicates the CPU cost of starting a Golang goroutine in TiDB. This variable is internally used in the [Cost Model](/cost-model.md).
```
Comment (TomShawn) [inline system-variables.md:1784]:
```suggestion
- Indicates the CPU cost of starting a Golang goroutine in TiDB. This variable is internally used in the [Cost Model](/cost-model.md), and it is not recommended to modify its value.
```

File: system-variables.md [hunk 14]
```diff
@@ -1724,6 +1764,34 @@ For a system upgraded to v5.0 from an earlier version, if you have not modified
...
+
+### tidb_opt_cop_cpu_factor
+
+- Scope: SESSION | GLOBAL
+- Persists to cluster: YES
+- Type: Float
+- Range: `[0, 2147483647]`
+- Default value: `3.0`
+- Indicates the CPU cost for TiKV Coprocessor to process one row. This variable is internally used in the [Cost Model](/cost-model.md).
```
Comment (TomShawn) [inline system-variables.md:1793]:
```suggestion
- Indicates the CPU cost for TiKV Coprocessor to process one row. This variable is internally used in the [Cost Model](/cost-model.md), and it is not recommended to modify its value.
```

File: system-variables.md [hunk 15]
```diff
@@ -1746,6 +1814,33 @@ For a system upgraded to v5.0 from an earlier version, if you have not modified
...
 - This variable is used to set the threshold value that determines whether to enable estimating the row count by using column order correlation. If the order correlation between the current column and the `handle` column exceeds the threshold value, this method is enabled.
 
+### tidb_opt_cpu_factor
+
+- Scope: SESSION | GLOBAL
+- Persists to cluster: YES
+- Type: Float
+- Range: `[0, 2147483647]`
+- Default value: `3.0`
+- Indicates the CPU cost for TiDB to process one row. This variable is internally used in the [Cost Model](/cost-model.md).
```
Comment (TomShawn) [inline system-variables.md:1824]:
```suggestion
- Indicates the CPU cost for TiDB to process one row. This variable is internally used in the [Cost Model](/cost-model.md), and it is not recommended to modify its value.
```

File: system-variables.md [hunk 16]
```diff
@@ -1746,6 +1814,33 @@ For a system upgraded to v5.0 from an earlier version, if you have not modified
...
+
+### tidb_opt_desc_scan_factor
+
+- Scope: SESSION | GLOBAL
+- Persists to cluster: YES
+- Type: Float
+- Range: `[0, 2147483647]`
+- Default value: `3.0`
+- Indicates the cost for TiKV to scan one row from the disk in descending order. This variable is internally used in the [Cost Model](/cost-model.md).
```
Comment (TomShawn) [inline system-variables.md:1833]:
```suggestion
- Indicates the cost for TiKV to scan one row from the disk in descending order. This variable is internally used in the [Cost Model](/cost-model.md), and it is not recommended to modify its value.
```

File: system-variables.md [hunk 17]
```diff
@@ -1746,6 +1814,33 @@ For a system upgraded to v5.0 from an earlier version, if you have not modified
...
+
+### tidb_opt_disk_factor
+
+- Scope: SESSION | GLOBAL
+- Persists to cluster: YES
+- Type: Float
+- Range: `[0, 2147483647]`
+- Default value: `1.5`
+- Indicates the I/O cost for TiDB to read or write one byte of data from or to the temporary disk. This variable is internally used in the [Cost Model](/cost-model.md).
```
Comment (TomShawn) [inline system-variables.md:1842]:
```suggestion
- Indicates the I/O cost for TiDB to read or write one byte of data from or to the temporary disk. This variable is internally used in the [Cost Model](/cost-model.md), and it is not recommended to modify its value.
```

File: system-variables.md [hunk 18]
```diff
@@ -1825,6 +1920,15 @@ mysql> desc select count(distinct a) from test.t;
...
 - If the value of the Limit or TopN operator is smaller than or equal to this threshold, these operators are forcibly pushed down to TiKV. This variable resolves the issue that the Limit or TopN operator cannot be pushed down to TiKV partly due to wrong estimation.
 
+### tidb_opt_memory_factor
+
+- Scope: SESSION | GLOBAL
+- Persists to cluster: YES
+- Type: Float
+- Range: `[0, 2147483647]`
+- Default value: `0.001`
+- Indicates the memory cost for TiDB to store one row. This variable is internally used in the [Cost Model](/cost-model.md).
```
Comment (TomShawn) [inline system-variables.md:1930]:
```suggestion
- Indicates the memory cost for TiDB to store one row. This variable is internally used in the [Cost Model](/cost-model.md), and it is not recommended to modify its value.
```

File: system-variables.md [hunk 19]
```diff
@@ -1833,6 +1937,15 @@ mysql> desc select count(distinct a) from test.t;
...
 - Specifies whether to allow using outer table as the build side in outer join for TiFlash.
 
+### tidb_opt_network_factor
+
+- Scope: SESSION | GLOBAL
+- Persists to cluster: YES
+- Type: Float
+- Range: `[0, 2147483647]`
+- Default value: `1.0`
+- Indicates the net cost of transferring 1 byte of data through the network. This variable is internally used in the [Cost Model](/cost-model.md).
```
Comment (TomShawn) [inline system-variables.md:1947]:
```suggestion
- Indicates the net cost of transferring 1 byte of data through the network. This variable is internally used in the [Cost Model](/cost-model.md), and it is not recommended to modify its value.
```

File: system-variables.md [hunk 20]
```diff
@@ -1874,6 +1987,24 @@ explain select * from t where age=5;
...
 - Specifies whether to allow the optimizer to push `Projection` down to the TiKV or TiFlash coprocessor.
 
+### tidb_opt_scan_factor
+
+- Scope: SESSION | GLOBAL
+- Persists to cluster: YES
+- Type: Float
+- Range: `[0, 2147483647]`
+- Default value: `1.5`
+- Indicates the cost for TiKV to scan one row of data from the disk in ascending order. This variable is internally used in the [Cost Model](/cost-model.md).
```
Comment (TomShawn) [inline system-variables.md:1997]:
```suggestion
- Indicates the cost for TiKV to scan one row of data from the disk in ascending order. This variable is internally used in the [Cost Model](/cost-model.md), and it is not recommended to modify its value.
```

File: system-variables.md [hunk 21]
```diff
@@ -1874,6 +1987,24 @@ explain select * from t where age=5;
...
+
+### tidb_opt_seek_factor
+
+- Scope: SESSION | GLOBAL
+- Persists to cluster: YES
+- Type: Float
+- Range: `[0, 2147483647]`
+- Default value: `20`
+- Indicates the start-up cost for TiDB to request data from TiKV. This variable is internally used in the [Cost Model](/cost-model.md).
```
Comment (TomShawn) [inline system-variables.md:2006]:
```suggestion
- Indicates the start-up cost for TiDB to request data from TiKV. This variable is internally used in the [Cost Model](/cost-model.md), and it is not recommended to modify its value.
```

**Review by TomShawn — APPROVED** [review]
LGTM

File: system-variables.md [hunk 22]
```diff
@@ -335,6 +335,38 @@ This variable is an alias for `last_insert_id`.
...
 > Unlike in MySQL, the `max_execution_time` system variable currently works on all kinds of statements in TiDB, not only restricted to the `SELECT` statement. The precision of the timeout value is roughly 100ms. This means the statement might not be terminated in accurate milliseconds as you specify.
 
+### max_prepared_stmt_count
+
+- Scope: GLOBAL
+- Persists to cluster: Yes
+- Type: Integer
+- Default value: `-1`
+- Range: `[-1, 1048576]`
+- Specifies the maximum number of [`PREPARE`](/sql-statements/sql-statement-prepare.md)  statements in a session.
```
Comment (TomShawn) [inline system-variables.md:345]:
```suggestion
- Specifies the maximum number of [`PREPARE`](/sql-statements/sql-statement-prepare.md) statements in a session.
```

File: system-variables.md [hunk 23]
```diff
@@ -1724,6 +1764,35 @@ For a system upgraded to v5.0 from an earlier version, if you have not modified
...
 - When the aggregate operation is slow in query, you can set the variable value to ON.
 
+### tidb_opt_cartesian_bcj
+
+- Scope: SESSION | GLOBAL
+- Persists to cluster: YES
+- Type: Integer
+- Default value: `1`
+- Range: `[0, 2]`
+- Indicates whether to allow the broadcast CARTESIAN join.
```
Comment (TomShawn) [inline system-variables.md:1774]:
```suggestion
- Indicates whether to allow the Broadcast CARTESIAN JOIN. 
```

File: system-variables.md [hunk 24]
```diff
@@ -1724,6 +1764,35 @@ For a system upgraded to v5.0 from an earlier version, if you have not modified
...
 
+### tidb_opt_cartesian_bcj
+
+- Scope: SESSION | GLOBAL
+- Persists to cluster: YES
+- Type: Integer
+- Default value: `1`
+- Range: `[0, 2]`
+- Indicates whether to allow the broadcast CARTESIAN join. 
+- `0` means that the broadcast CARTESIAN join is not allowed. `1` means that it is allowed based on [`tidb_broadcast_join_threshold_count`](#tidb_broadcast_join_threshold_count-new-in-v50). `2` means that it is always allowed even if the table size exceeds the threshold.
```
Comment (TomShawn) [inline system-variables.md:1775]:
```suggestion
- `0` means that the Broadcast Cartesian Join is not allowed. `1` means that it is allowed based on [`tidb_broadcast_join_threshold_count`](#tidb_broadcast_join_threshold_count-new-in-v50). `2` means that it is always allowed even if the table size exceeds the threshold.
```

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

File: system-variables.md [hunk 25]
```diff
@@ -1746,6 +1815,33 @@ For a system upgraded to v5.0 from an earlier version, if you have not modified
...
 - This variable is used to set the threshold value that determines whether to enable estimating the row count by using column order correlation. If the order correlation between the current column and the `handle` column exceeds the threshold value, this method is enabled.
 
+### tidb_opt_cpu_factor
+
+- Scope: SESSION | GLOBAL
+- Persists to cluster: YES
+- Type: Float
+- Range: `[0, 2147483647]`
+- Default value: `3.0`
+- Indicates the CPU cost for TiDB to process one row. This variable is internally used in the [Cost Model](/cost-model.md), and it is not recommended to modify its value.
```
Comment (TomShawn) [inline system-variables.md:1825]:
```suggestion
- Indicates the CPU cost for TiDB to process one row. This variable is internally used in the [Cost Model](/cost-model.md), and it is **NOT** recommended to modify its value.
```

File: system-variables.md [hunk 26]
```diff
@@ -1746,6 +1815,33 @@ For a system upgraded to v5.0 from an earlier version, if you have not modified
...
+
+### tidb_opt_desc_scan_factor
+
+- Scope: SESSION | GLOBAL
+- Persists to cluster: YES
+- Type: Float
+- Range: `[0, 2147483647]`
+- Default value: `3.0`
+- Indicates the cost for TiKV to scan one row from the disk in descending order. This variable is internally used in the [Cost Model](/cost-model.md), and it is not recommended to modify its value.
```
Comment (TomShawn) [inline system-variables.md:1834]:
```suggestion
- Indicates the cost for TiKV to scan one row from the disk in descending order. This variable is internally used in the [Cost Model](/cost-model.md), and it is **NOT** recommended to modify its value.
```

File: system-variables.md [hunk 27]
```diff
@@ -1746,6 +1815,33 @@ For a system upgraded to v5.0 from an earlier version, if you have not modified
...
+
+### tidb_opt_disk_factor
+
+- Scope: SESSION | GLOBAL
+- Persists to cluster: YES
+- Type: Float
+- Range: `[0, 2147483647]`
+- Default value: `1.5`
+- Indicates the I/O cost for TiDB to read or write one byte of data from or to the temporary disk. This variable is internally used in the [Cost Model](/cost-model.md), and it is not recommended to modify its value.
```
Comment (TomShawn) [inline system-variables.md:1843]:
```suggestion
- Indicates the I/O cost for TiDB to read or write one byte of data from or to the temporary disk. This variable is internally used in the [Cost Model](/cost-model.md), and it is **NOT** recommended to modify its value.
```

File: system-variables.md [hunk 28]
```diff
@@ -1825,6 +1921,15 @@ mysql> desc select count(distinct a) from test.t;
...
 - If the value of the Limit or TopN operator is smaller than or equal to this threshold, these operators are forcibly pushed down to TiKV. This variable resolves the issue that the Limit or TopN operator cannot be pushed down to TiKV partly due to wrong estimation.
 
+### tidb_opt_memory_factor
+
+- Scope: SESSION | GLOBAL
+- Persists to cluster: YES
+- Type: Float
+- Range: `[0, 2147483647]`
+- Default value: `0.001`
+- Indicates the memory cost for TiDB to store one row. This variable is internally used in the [Cost Model](/cost-model.md), and it is not recommended to modify its value.
```
Comment (TomShawn) [inline system-variables.md:1931]:
```suggestion
- Indicates the memory cost for TiDB to store one row. This variable is internally used in the [Cost Model](/cost-model.md), and it is **NOT** recommended to modify its value.
```

File: system-variables.md [hunk 29]
```diff
@@ -1833,6 +1938,15 @@ mysql> desc select count(distinct a) from test.t;
...
 - Specifies whether to allow using outer table as the build side in outer join for TiFlash.
 
+### tidb_opt_network_factor
+
+- Scope: SESSION | GLOBAL
+- Persists to cluster: YES
+- Type: Float
+- Range: `[0, 2147483647]`
+- Default value: `1.0`
+- Indicates the net cost of transferring 1 byte of data through the network. This variable is internally used in the [Cost Model](/cost-model.md), and it is not recommended to modify its value.
```
Comment (TomShawn) [inline system-variables.md:1948]:
```suggestion
- Indicates the net cost of transferring 1 byte of data through the network. This variable is internally used in the [Cost Model](/cost-model.md), and it is **NOT** recommended to modify its value.
```

File: system-variables.md [hunk 30]
```diff
@@ -1874,6 +1988,24 @@ explain select * from t where age=5;
...
 - Specifies whether to allow the optimizer to push `Projection` down to the TiKV or TiFlash coprocessor.
 
+### tidb_opt_scan_factor
+
+- Scope: SESSION | GLOBAL
+- Persists to cluster: YES
+- Type: Float
+- Range: `[0, 2147483647]`
+- Default value: `1.5`
+- Indicates the cost for TiKV to scan one row of data from the disk in ascending order. This variable is internally used in the [Cost Model](/cost-model.md), and it is not recommended to modify its value.
```
Comment (TomShawn) [inline system-variables.md:1998]:
```suggestion
- Indicates the cost for TiKV to scan one row of data from the disk in ascending order. This variable is internally used in the [Cost Model](/cost-model.md), and it is **NOT** recommended to modify its value.
```

File: system-variables.md [hunk 31]
```diff
@@ -1874,6 +1988,24 @@ explain select * from t where age=5;
...
+
+### tidb_opt_seek_factor
+
+- Scope: SESSION | GLOBAL
+- Persists to cluster: YES
+- Type: Float
+- Range: `[0, 2147483647]`
+- Default value: `20`
+- Indicates the start-up cost for TiDB to request data from TiKV. This variable is internally used in the [Cost Model](/cost-model.md), and it is not recommended to modify its value.
```
Comment (TomShawn) [inline system-variables.md:2007]:
```suggestion
- Indicates the start-up cost for TiDB to request data from TiKV. This variable is internally used in the [Cost Model](/cost-model.md), and it is **NOT** recommended to modify its value.
```

File: system-variables.md [hunk 32]
```diff
@@ -1724,6 +1764,35 @@ For a system upgraded to v5.0 from an earlier version, if you have not modified
 - This variable is used to set whether the optimizer executes the optimization operation of pushing down the aggregate function to the position before Join, Projection, and UnionAll.
 - When the aggregate operation is slow in query, you can set the variable value to ON.
 
+### tidb_opt_cartesian_bcj
```
Comment (TomShawn) [inline system-variables.md:1767]: Incorrect variable name. Fixed in https://github.com/pingcap/docs/pull/10782

File: system-variables.md [hunk 33]
```diff
@@ -1724,6 +1764,35 @@ For a system upgraded to v5.0 from an earlier version, if you have not modified
...
+- Scope: SESSION | GLOBAL
+- Persists to cluster: YES
+- Type: Integer
+- Default value: `1`
+- Range: `[0, 2]`
+- Indicates whether to allow the Broadcast Cartesian Join. 
+- `0` means that the Broadcast Cartesian Join is not allowed. `1` means that it is allowed based on [`tidb_broadcast_join_threshold_count`](#tidb_broadcast_join_threshold_count-new-in-v50). `2` means that it is always allowed even if the table size exceeds the threshold.
+- This variable is internally used in TiDB, and it is **NOT** recommended to modify its value.
+
+### tidb_opt_concurrency_factor
```
Comment (TomShawn) [inline system-variables.md:1778]: Incorrect variable name. Fixed in https://github.com/pingcap/docs/pull/10782

File: system-variables.md [hunk 34]
```diff
@@ -1724,6 +1764,35 @@ For a system upgraded to v5.0 from an earlier version, if you have not modified
...
+
+- Scope: SESSION | GLOBAL
+- Persists to cluster: YES
+- Type: Float
+- Range: `[0, 2147483647]`
+- Default value: `3.0`
+- Indicates the CPU cost of starting a Golang goroutine in TiDB. This variable is internally used in the [Cost Model](/cost-model.md), and it is **NOT** recommended to modify its value.
+
+### tidb_opt_cop_cpu_factor
```
Comment (TomShawn) [inline system-variables.md:1787]: Incorrect variable name. Fixed in https://github.com/pingcap/docs/pull/10782

File: system-variables.md [hunk 35]
```diff
@@ -1833,6 +1938,15 @@ mysql> desc select count(distinct a) from test.t;
 - Default value: `ON`
```
Comment (TomShawn) [inline system-variables.md:1938]: Incorrect default value. Fixed in https://github.com/pingcap/docs/pull/10782.

**Pull request merged**